python/adelphi/bin/adelphi

```diff
@@ -30,6 +34,20 @@ try:
 except ImportError:
     from backports.tempfile import TemporaryDirectory
 
+try:
+    from datetime import timezone
+    utc = timezone.utc
+except ImportError:
+    class UTC(tzinfo):
+        def utcoffset(self, dt):
+            return timedelta(0)
+        def tzname(self, dt):
+            return "UTC"
+        def dst(self, dt):
+            return timedelta(0)
+    utc = UTC()
+
+
```

Comment:
Another Python2/3 incompatibility.  Python 2.x doesn't have datetime.timestamp.utc so we make a fake one if we can't find one provided by the platform.  Impl here based on https://stackoverflow.com/questions/7065164/how-to-make-an-unaware-datetime-timezone-aware-in-python